app/views/users/show.html.erb

```diff
@@ -8,4 +8,13 @@
       </h1>
     </section>
   </aside>
+  <div class="col-md-8">
+    <% if @user.microposts.any? %>
+      <h3><%= t(".microposts") %> (<%= @user.microposts.count %>)</h3>
```

Comment:
so sánh count, size, length

Reply:
count: Đếm số phần tử bằng cách sử dụng truy vấn SQL (SELECT COUNT(*) FROM...). mang ý nghĩa là số phần tử trong database
size: sẽ vào database gọi đến SQL SELECT COUNT(*) khi chưa có cache (tức là khi chưa có các bản ghi users trên bộ nhớ tạm), khi đã có rồi thì đếm luôn số bản ghi trên bộ nhớ tạm.
length: sẽ tính toán độ dài của mảng kết quả, mang ý nghĩa là số phần tử trên mảng

Reply:
dùng size đi e